src/jobsautoreport/slack.py

```diff
@@ -0,0 +1,131 @@
+import logging
+from typing import Optional
+
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+logger = logging.getLogger(__name__)
+
+
+class SlackMessage:
+    def __init__(self, channel_id: str, data: dict) -> None:
+        self.channel_id = channel_id
+        self.data = data
+        self.ts = None
+
+    def format_as_message(self) -> list:
```

Comment:
maybe even 
```suggestion
    def format_as_message(self) -> List[Dict[str, str]]:
```

Reply:
`List`, `Dict` and some others from `typing` module are deprecated:

```
Deprecated since version 3.9: [builtins.dict](https://docs.python.org/3/library/stdtypes.html#dict) now supports subscripting ([]). See [PEP 585](https://peps.python.org/pep-0585/) and [Generic Alias Type](https://docs.python.org/3/library/stdtypes.html#types-genericalias).
```

https://docs.python.org/3/library/typing.html#generic-concrete-collections

Reply:
coolz! I didn't know that

Reply:
I think that means that the value in `dict` must be a `string` isn't it? here it is also a dict
Maybe `list[dict]`